slog/slog.go

```diff
@@ -114,6 +119,32 @@ func Configure(cfg Config) error {
 	return nil
 }
 
+// Customize the http request fields
+// More: https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#HttpRequest
+func convertHTTPRequest(orig slog.Value) (string, slog.Value) {
+	var attrs []slog.Attr
+	origMap, _ := orig.Any().(map[string]any)
```

Comment:
Can't you just manipulate origMap, copying field X to Y like:

```go
origMap["requestMethod"] = origMap["method"]
delete(origMap, "method")
```

I imagine the mutations on this underlying map will be reflected on the given slog.Value no ? (maps are very side effecty =P...and I doubt that Any() returns a deep copy).

Reply:
Not completely against doing a full copy...specially if it is just per request..should not be a huge problem....it just seems like a no copy solution should be feasible here.

Reply:
> Can't you just manipulate origMap, copying field X to Y like:
Yes, I started doing that... but indeed happen what you mention: we have 2 lines the "handling" and the "handled" reusing the same map, so if I delete one it affects the other. I could create different maps to workaround that.

I'm not really sure which approach is less problematic. If you're ok with copying I can keep it.

Reply:
we can start with copying and if it is a problem we re-assess...I was just wondering why it would not be possible..because it seems possible =P